tests/MSBuildDeviceIntegration/Tests/MonoAndroidExportTest.cs

```diff
@@ -135,8 +135,9 @@ protected override void OnCreate (Bundle bundle)
 </manifest>";
 				Assert.True (b.Install (proj), "Project should have installed.");
 				ClearAdbLogcat ();
+				b.BuildLogFile = "run.log";
 				if (CommercialBuildAvailable)
-					Assert.True (b.RunTarget (proj, "_Run"), "Project should have run.");
+					Assert.True (b.RunTarget (proj, "_Run", doNotCleanupOnUpdate: true), "Project should have run.");
```

Comment:
Should we redo code like this so they use `StartAndroidActivity` instead?

https://github.com/xamarin/xamarin-android/blob/77464dc92ee53a5c0ef62b1df78d4e2bb4af5992/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Application.targets#L15

Reply:
Well the StartAndroidActivity will work in some cases, but it does not call the internal "_Run" target which sets up the debugger so we can't use it everywhere.